website/docs/r/atracker_route.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_atracker_route"
+description: |-
+  Manages Activity Tracking Route.
+subcategory: "Activity Tracking API"
+---
+
+# ibm_atracker_route
+
+Provides a resource for Activity Tracking Route. This allows Activity Tracking Route to be created, updated and deleted.
+
+## Example Usage
+
+```hcl
+resource "ibm_atracker_route" "atracker_route" {
+  name = "my-route"
+  receive_global_events = false
+  rules = { "target_ids" : [ "target_ids" ] }
```

Comment:
This syntax of rules in invalid we can't have = symbol
```
rules {
   target_ids = [ ibm_atracker_target.atracker_target.id ]
}
```

Reply:
fixed